xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/Panels/Applications.xml

```diff
@@ -649,15 +659,15 @@ require(['scriptaculous/effects'], function() {
 .panel-width-Large .applicationsPanel &gt; li &gt; a:focus,
 .panel-width-Large .applicationsPanelMoreList &gt; li &gt; a:hover,
 .panel-width-Large .applicationsPanelMoreList &gt; li &gt; a:focus,
-.panel-width-Large .applicationPanelMoreContainer ul &gt; li &gt; a:hover 
+.panel-width-Large .applicationPanelMoreContainer ul &gt; li &gt; a:hover
```

Comment:
Why is there no `,` at the end of the line?

Reply:
The diff is weird :thinking: , it looks like they weren't any beforehand too, just a space.
Maybe it was a bug but it got never reported because I don't think many people use large panels, and it's not an UI breaking bug, just a couple of colors (maybe) looking weird on anchors. I don't think we have a manual test covering this.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2204/commits/aaf04ab737288a251968f37ab39fe816f8dc8000